api.js

```diff
@@ -56,7 +61,17 @@ Api.prototype._runFile = function (file) {
 		args.push('--sorted');
```

Comment:
Worth switching to [execa](https://github.com/sindresorhus/execa) at the same time?